lib/t/push_notifications/dispatch_job.ex

```diff
@@ -190,6 +190,7 @@ defmodule T.PushNotifications.DispatchJob do
   defp profile_info(user_id) do
     Accounts.Profile
     |> where(user_id: ^user_id)
+    |> select([p], {p.name, p.gender})
```

Comment:
but with this the the extraction should be different, this change alone wouldn't work

previously you fetched the full %Profile{} struct and extracted gender and name like this:
![image](https://user-images.githubusercontent.com/16062322/142440675-75648b31-8686-4239-8dfa-d80fcd88f210.png)

but now you are not selecting the whole Profile struct, you are extracting just two variables in a tuple {name, gender}, therefore the extraction should be similar, something like:
{name, gender} = profile_info(user_id)
and
{name, _gender} = profile_info(user_id) - in cases where you only need names, don't forget to change them as well

Reply:
![image](https://user-images.githubusercontent.com/16062322/142442969-6a18eac1-8b9e-410a-8c74-b6bd3bc90f95.png)

Reply:
thanks for good review, changed using of struct like you said